src/org/usfirst/frc4904/robot/Robot.java

```diff
@@ -71,6 +75,11 @@ public void testInitialize() {}
 	@Override
 	public void testExecute() {}
 
+	byte[] matchConfigurationCANData() {
+		return new byte[] {(byte) DriverStation.getInstance().getAlliance().ordinal(),
```

Comment:
Is there a high degree of confidence that WPILib is consistent regarding the ordering of their alliances within the enum?